<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GoogleSettingsDialog</class>
 <widget class="QWidget" name="GoogleSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>680</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="accountTextLabel">
       <property name="text">
        <string>Account:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="accountLabel">
       <property name="text">
        <string>&lt;b&gt;not configured&lt;/b&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="configureBtn">
       <property name="text">
        <string>Configure...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="refreshBox">
     <property name="title">
      <string>Refresh</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="refreshLabel">
        <property name="text">
         <string>Refresh interval:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>refreshSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KPluralHandlingSpinBox" name="refreshSpinBox">
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>720</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="enableRefresh">
        <property name="text">
         <string>Enable interval refresh</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="calendarsBox">
     <property name="title">
      <string>Calendars</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QListWidget" name="calendarsList"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="eventsLimitLabel">
        <property name="text">
         <string>Fetch only events since</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>eventsLimitCombo</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="KDateComboBox" name="eventsLimitCombo"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="reloadCalendarsBtn">
        <property name="text">
         <string>Reload</string>
        </property>
        <property name="icon">
         <iconset theme="view-refresh">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="taskListsBox">
     <property name="title">
      <string>Tasklists</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="taskListsList"/>
      </item>
      <item>
       <widget class="QPushButton" name="reloadTaskListsBtn">
        <property name="text">
         <string>Reload</string>
        </property>
        <property name="icon">
         <iconset theme="view-refresh">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPluralHandlingSpinBox</class>
   <extends>QSpinBox</extends>
   <header>kpluralhandlingspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>KDateComboBox</class>
   <extends>QComboBox</extends>
   <header>kdatecombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>enableRefresh</sender>
   <signal>toggled(bool)</signal>
   <receiver>refreshSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
